systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaBridgeUtils.java

```diff
@@ -64,11 +68,40 @@ public static void checkSendResponse(JsonObject response, int messageCount) {
         }
     }
 
-    public static void waitUntilKafkaBridgeStatus(String clusterName, String state) {
+    /**
+     * Wait until KafkaBridge is in desired state
+     * @param clusterName name of KafkaBridge cluster
+     * @param state desired state
+     */
+    public static void waitForKafkaBridgeStatus(String clusterName, String state) {
         LOGGER.info("Waiting till Kafka Bridge CR will be in state: {}", state);
         TestUtils.waitFor("Waiting for Kafka resource status is: " + state, Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
-            () -> KafkaBridgeResource.kafkaBridgeClient().inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getStatus().getConditions().get(0).getType().equals(state)
+            () -> kafkaBridgeClient().inNamespace(namespace).withName(clusterName).get().getStatus().getConditions().get(0).getType().equals(state),
+            () -> StUtils.logCurrentStatus(kafkaBridgeClient().inNamespace(namespace).withName(clusterName).get())
         );
         LOGGER.info("Kafka Bridge CR is in state: {}", state);
     }
+
+    public static void waitForKafkaBridgeReady(String clusterName) {
+        waitForKafkaBridgeStatus(clusterName, "Ready");
+    }
+
+    public static void waitForKafkaBridgeNotReady(String clusterName) {
+        waitForKafkaBridgeStatus(clusterName, "NotReady");
+    }
+
+    /**
+     * Wait until KafkaBridge and its pods will be in Ready state
+     * @param clusterName name of KafkaBridge cluster
+     * @param expectPods number of expected pods to be ready
+     */
+    public static void waitForKafkaBridgeReady(String clusterName, int expectPods) {
```

Comment:
You seem to have two methods `waitForKafkaBridgeReady` and one `waitForKafkaBridgeNotReady`. But each works a bit differently -> one checks the pods and KafkaBridge status and one checks the CR status only. Wouldn't that be confusing when using it?

Reply:
The `waitForKafkaBridgeReady` and `waitForKafkaBridgeNotReady` are only overloading methods so we don't need always write `"Ready"` or `"NotReady"` states for each wait and use it little bit easier. 

About second part of this nit - when I took look on `DeploymentUtils` (as lot of our CR deploy used this class) we have there two same name methods `waitForDeploymentReady` - one for only deployment to be ready and the second one for deployment **and** pods to be ready. Maybe it's just opinion but for me it's not confusing. Because if you want to check also if pods are ready, you'll just add number of replicas to method parameters.

We discuss this with @see-quick and @Frawless sooner. I think we can change it to be more clear (@see-quick has this comment too).